### ISSUE TITLE
PSX for macOS ?

### ISSUE THREAD

**Issue by Momeks** [KOPElan/PSX-Download-Helper#1]
Hi, thank for this great app. Is there any chance to release this app for macOS? We really need it in mac. Please please please

**Comment by FurryR** [comment]
use boot camp

**Comment by aliab** [comment]
I'm on Apple silicon

**Comment by mortezaghafoori** [comment]
Yeah, anyone has any ideas on how to run this app on apple silicon macs?
